### PR TITLE
Trips

### PR DESCRIPTION
**Pull request by elagin**
Save trips into database. Make some statistics.

P.S. Not a very elegant solution in updateTripStatistic.

### CODE REVIEW

File: src/main/java/pt/lighthouselabs/obd/reader/activity/ConfirmDialog.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+public class ConfirmDialog {
+
+  public static final int DIALOG_CONFIRM_DELETE_ID = 1;
+
+  public static Dialog createDialog(final int id, Context context, final Listener listener) {
+    String title;
+    String message;
+
+    switch (id) {
+      case DIALOG_CONFIRM_DELETE_ID: // Диалоговое окно About
```
Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/reader/activity/ConfirmDialog.java]: Can you please use english for comments as well?

File: src/main/java/pt/lighthouselabs/obd/reader/activity/MainActivity.java [hunk 2]
```diff
@@ -315,10 +356,9 @@ protected void onResume() {
     final BluetoothAdapter btAdapter = BluetoothAdapter
         .getDefaultAdapter();
 
-    preRequisites = btAdapter != null && btAdapter.isEnabled();
-    if ( !preRequisites && prefs.getBoolean(ConfigActivity.ENABLE_BT_KEY, false)) {
-         preRequisites = btAdapter.enable();
-    }
```
Comment (nomwerp) [inline src/main/java/pt/lighthouselabs/obd/reader/activity/MainActivity.java]: was this intended, it seems to revert a recent commit?

Comment (pires) [inline src/main/java/pt/lighthouselabs/obd/reader/activity/MainActivity.java]: Good catch!

**Comment by elagin** [comment]
I think I need to apply this request, and only then to discuss how to use it in conjunction with sending data.

**Comment by nomwerp** [comment]
yes agreed

**Comment by pires** [comment]
@elagin LGTM. Can I merge?

**Comment by elagin** [comment]
Yes, of course.

**Comment by pires** [comment]
Thank you @elagin. You and @nomwerp have brought a new life to this project.

**Pull request merged**